### PR TITLE
animación Homepage V1

### PR DESCRIPTION
**Pull request by Felipe34515**
Hacer la primera versión de la HomePage con animación en el código.
image

### CODE REVIEW

File: src/layouts/Layout.astro [hunk 1]
```diff
@@ -4,48 +4,29 @@ interface Props {
...
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="description" content="Astro description" />
 		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@400;700&display=swap">
+		<link rel="icon"  type="image/svg+xml" href="https://avatars.githubusercontent.com/u/127456648?s=200&v=4" />
 		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
+		<title>Computer Society</title>
```
Comment (Crola1702) [inline src/layouts/Layout.astro:20]: I think it's not a good idea to hardcode the title here. Please revert this change and add your title as a prop.

**Review by Crola1702 — CHANGES_REQUESTED** [review]
Just one minor comment

**Review by Crola1702 — APPROVED** [review]
LGTM

**Comment by Crola1702** [comment]
After solving conflicts

**Comment by mr-sosa** [comment]
Finally, don't forget resolved the merge conflicts. You must leave the index.astro of your branch as it's in the branch develop

**Pull request merged**